apps/site/next.helpers.mjs

```diff
@@ -53,8 +40,27 @@ export const getMarkdownFiles = async (root, cwd, ignore = []) => {
   const cacheKey = `${root}${cwd}${ignore.join('')}`;
 
   if (!globCacheByPath.has(cacheKey)) {
-    globCacheByPath.set(cacheKey, glob('**/*.{md,mdx}', { root, cwd, ignore }));
+    const keyFiles = files
```

Comment:
This is hacky as hell and complex as hell. Please let's not do this.

Reply:
yeah definitely... but as I mentioned in [my other comment](https://github.com/nodejs/nodejs.org/pull/7383#discussion_r1901974649) I think that currently we can't really do much better than this (besides cleaning it up and move it in a less hacky way, etc...), since we cannot really run `glob()` in a deployed Worker...

(basically I would cathegorize this as the same issue as https://github.com/nodejs/nodejs.org/pull/7383#discussion_r1901922705)

Reply:
outdated, I've removed all the generated files hacks 🚀